src/nvim/api/buffer.c

```diff
@@ -1437,6 +1437,9 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id,
 ///                              default
 ///                 - "combine": combine with background text color
 ///                 - "blend": blend with background text color.
+///               - hl_eol : when a multiline highlight covers the EOL of a line
+///                          continue the highlight for the rest of the screen
+///                          line, just like for diff and cursorline highlight
```

Comment:
This adds the type, and a comma after `...EOL of a line`, and re-flows the text
```suggestion
///               - hl_eol : boolean indicating that when a multiline highlight
///                          covers the EOL of a line, continue the highlight for
///                          the rest of the screen line.
///                          (just like for diff and cursorline highlights)
```